webpack/InsightsHostDetailsTab/NewHostDetailsTab.js

```diff
@@ -104,4 +108,50 @@ NewHostDetailsTab.defaultProps = {
   router: {},
 };
 
-export default NewHostDetailsTab;
+// Local Insights advisor
+const scope = 'advisor';
+// eslint-disable-next-line spellcheck/spell-checker
+const module = './HostDetailsLightspeedTabWrapped';
+
+const IopInsightsTab = props => (
+  <ScalprumComponent
+    scope={scope}
+    module={module}
+    IopRemediationModal={RemediationModal}
+    generateRuleUrl={generateRuleUrl}
+    {...props}
+  />
+);
+
+const IopInsightsTabWrapped = props => (
+  <ScalprumProvider {...providerOptions}>
+    <IopInsightsTab {...props} />
+  </ScalprumProvider>
+);
+
+const LightspeedTab = props => {
```

Comment:
Could you do this here?
```suggestion
const LightspeedTab = ({ response, ...props }) => {
```